core/src/main/java/io/questdb/log/Logger.java

```diff
@@ -358,7 +358,7 @@ private LogRecord next(Sequence seq, RingQueue<LogRecordSink> ring, int level) {
             return NullLogRecord.INSTANCE;
         }
 
-        final long cursor = seq.next();
+        final long cursor = seq.nextBully();
```

Comment:
@bluestreak01 I don't think this change comes from my commit. Looks like a debug leftover. WDYT?

Reply:
yeah, its mine, sync logger, will remove